frontend/src/store/models/user.ts

```diff
@@ -42,5 +51,61 @@ export const user = createModel<RootModel>()({
         console.log(err);
       }
     },
+
+    async signIn({ username, password }) {
+      try {
+        const response = await http.post(SIGN_IN_URL, {
+          username,
+          password,
+        });
+        dispatch.user.setIsAuthenticated({ isAuthenticated: true });
+        dispatch.user.setTokens({
+          accessToken: response.data.accessToken,
+          refreshToken: response.data.refreshToken,
+        });
+        setUserTokensToLocalStorage(response.data.accessToken, response.data.refreshToken);
+      } catch (error) {
+        console.log(error);
+      }
+    },
+
+    async signOut() {
+      const accessToken = getAccessToken();
+      await http.post(
+        SIGN_OUT_URL,
+        {},
+        {
+          headers: {
+            Authorization: `Bearer ${accessToken}`,
+          },
+        }
```

Comment:
I think we should write wrapper for request to write it only one time

Reply:
yeah, you are right